### ISSUE TITLE
restart cyclegan

### ISSUE THREAD

**Issue by chenhaibin2019** [junyanz/pytorch-CycleGAN-and-pix2pix#448]
I want to restart the training from the latest saved checkpoint. (epoch 200) see if the generated image quality can get further improved.

I fun it from (~/CycleGAN/pytorch-CycleGAN-and-pix2pix$)
python train.py --epoch_count 201 --continue_train --dataroot ./datasets/d400 --name d400 --model cycle_gan --batch_size 1 --gpu_ids 5 --display_id -1

There is existed d400 folder in checkpoint from last training. However, the program didn't start training though print out the log information as below. Where I got it wrong? how to restart training? thank you

![image](https://user-images.githubusercontent.com/39716577/48788891-c8657f80-ecb9-11e8-8f54-8ffc4d229980.png)

**Comment by junyanz** [comment]
See this [Q & A](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/docs/qa.md#can-i-continueresume-my-training-350-275-234-87) for continue training.

**Comment by chenhaibin2019** [comment]
thanks
it seems --epoch_count 200 restarts training while 201 does not. 
Another question: is the restart learning rate is default value 0.0002? Is there a recommended learning rate for restart?

**Comment by junyanz** [comment]
You probably need to set `--niter` and `--niter_decay`. See [here](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/options/train_options.py#L21),
